python/tests/test-1-vector.py

```diff
@@ -366,6 +343,30 @@ def test_125(ceed_resource, capsys):
     with y.array() as b:
         assert np.allclose(1.5 * a, b)
 
+# -------------------------------------------------------------------------------
+# Test vector copy
+# -------------------------------------------------------------------------------
+
+
+def test_126(ceed_resource, capsys):
+    ceed = libceed.Ceed(ceed_resource)
+
+    n = 10
+
+    x = ceed.Vector(n)
+    y = ceed.Vector(n)
+
+    a = np.arange(10, 10 + n, dtype=ceed.scalar_type())
+    x.set_array(a, cmode=libceed.USE_POINTER)
+
+    with x.array() as x_array:
+        y.copy(x_array)
```

Comment:
I think the test is upset because it wants a libCEED vector?
```
y.copy(x)
```
I wonder if it might make sense to replicate the Rust interface change?